utils/build-presets.ini

```diff
@@ -2386,11 +2386,9 @@ skip-test-cmark
 build-swift-tools=0
 skip-early-swift-driver
 
-# Then set the paths to our native tools. If compiling against a toolchain,
-# these should all be the ./usr/bin directory.
-native-swift-tools-path=%(toolchain_path)s
-native-llvm-tools-path=%(toolchain_path)s
-native-clang-tools-path=%(toolchain_path)s
+# Then set the paths to our native tools. If compiling against a toolchain, this
+# should point to the root of the toolchain.
+custom-native-toolchain=%(toolchain_path)s
```

Comment:
I've replaced the native flags with this one. It should be a drop-in replacement assuming that the tools all live in the same place, so even if it's the `<root>/bin` dir, it's fine.

@buttaface I'm not sure if this affects Android or how you have that set up. While I'm pretty sure it should Just Work™, I'd like to actually verify it. Do you have instructions for doing this? I'm on macOS and have docker, but don't have any android devices.

Reply:
I currently don't use any build presets, [specifying all the flags instead](https://github.com/buttaface/swift-android-sdk#building-the-android-sdks), so this change won't break anything for me.